web-client/.gitignore

```diff
@@ -11,6 +11,8 @@
 # next.js
 /.next/
 /out/
+.next
```

Comment:
Correct me if I'm wrong, but I think it's `.next` instead of `/.next/` that would actually exclude the next files from being added to github. Looks like some files have been added already.. No harm in keeping both in the gitignore

Reply:
No clue either way, didn't touch the file, so `.next` works for me too